WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -11,4 +24,82 @@ public static void showSitePickerForResult(Activity activity, boolean visibleAcc
         activity.startActivityForResult(intent, RequestCodes.SITE_PICKER);
     }
 
+    public static void viewCurrentSite(Activity activity) {
```

Comment:
For all of these that don't use `startActivityForResult` we should pass a context rather than an activity. This enables calling these routines from adapters and other places where we have a context but not necessarily an activity. [Example](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderActivityLauncher.java#L58)